### PR TITLE
Enable sound play flag

### PR DESCRIPTION
**Pull request by iory**
# What is this?

Add a flag to enable or disable sound to play sound on recognition and add `play_sound` for continuous recognition mode.

You can select mute or unmute sound effects on speech recognition.

## Quick start

Using test rosbag, you can try this PR.

```
roslaunch ros_speech_recognition sample_ros_speech_recognition.test launch_sound_play:=true
```

### CODE REVIEW

**Comment by mqcmd196** [comment]
I'm sorry that I'm not familiar with `ros_speech_recognition`. Is this PR for muting sound effect of `ros_speech_recognition` result?

File: ros_speech_recognition/README.md [hunk 1]
```diff
@@ -52,6 +52,10 @@ This package uses Python package [SpeechRecognition](https://pypi.python.org/pyp
 
 ## Parameters
 
+* `~enable_sound_play` (`Bool`, default: `True`)
```
Comment (mqcmd196) [inline ros_speech_recognition/README.md:55]: If https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/315#issuecomment-1001519577 is correct, I'd like change the param name to `~enable_sound_effect`, which is more understandable.

**Review by mqcmd196 — CHANGES_REQUESTED** [review]

File: ros_speech_recognition/README.md (hunk 1 above)
Comment (iory) [reply ros_speech_recognition/README.md]: Thanks. You're right.
I modified the param name.

**Comment by iory** [comment]
> Is this PR for muting sound effect of ros_speech_recognition result?

You're right. With this PR, we can select mute or unmute sound effects on speech recognition.

**Review by mqcmd196 — APPROVED** [review]
LGTM, test passed!

File: ros_speech_recognition/README.md (hunk 1 above)
Comment (mqcmd196) [reply ros_speech_recognition/README.md]: Thanks!

**Comment by mqcmd196** [comment]
I confirmed it works fine.

**Pull request merged**